client/src/app/auth/signin/page.tsx

```diff
@@ -11,5 +19,9 @@ export default async function SignInPage() {
     redirect("/profile");
   }
 
-  return <SignIn />;
+  return (
+    <AuthLayout className="space-y-2 pt-20">
```

Comment:
I am seeing the same classes over different pages, perhaps we can put them inside the `AuthLayout`?